common/persistence/elasticsearch/esProcessor.go

```diff
@@ -385,23 +392,9 @@ func getErrorMsgFromESResp(resp *elastic.BulkResponseItem) string {
 	return errMsg
 }
 
-func newChanWithStopwatch(ackCh chan<- bool, stopwatch *tally.Stopwatch) *chanWithStopwatch {
-	return &chanWithStopwatch{
+func newAckChanWithStopwatch(ackCh chan<- bool, stopwatch *tally.Stopwatch) *ackChanWithStopwatch {
+	return &ackChanWithStopwatch{
 		ackCh:             ackCh,
 		addToAckStopwatch: stopwatch,
 	}
 }
-
-func (km *chanWithStopwatch) Ack() {
-	km.ackCh <- true
-	if km.addToAckStopwatch != nil {
-		km.addToAckStopwatch.Stop()
-	}
-}
-
-func (km *chanWithStopwatch) Nack() {
-	km.ackCh <- false
-	if km.addToAckStopwatch != nil {
-		km.addToAckStopwatch.Stop()
-	}
-}
```

Comment:
not used?

Reply:
Yes, it is easier to do these two operations in place.